Code/Tools/TestImpactFramework/Runtime/Common/Code/Include/Headers/TestImpactFramework/TestImpactTestSequenceBus.h

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <TestImpactFramework/TestImpactClientSequenceReport.h>
+#include <TestImpactFramework/TestImpactClientTestRun.h>
+#include <TestImpactFramework/TestImpactClientTestSelection.h>
+#include <TestImpactFramework/TestImpactTestSequence.h>
+
+#include <AzCore/EBus/EBus.h>
+
+namespace TestImpact
+{
+    //! Base Bus for test sequence notifications.
+    class TestSequenceNotificationsBase
+        : public AZ::EBusTraits
+    {
+    public:
+        // EBusTraits overrides ...
+        static constexpr AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::Single;
+        static constexpr AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Multiple;
+
+        ~TestSequenceNotificationsBase() = default;
```

Comment:
Need to make this virtual?

Reply:
(see earlier comment - leave as non-virtual but make protected)

Reply:
Fixed 👍